rules/windows/persistence_webshell_detection.toml

```diff
@@ -21,7 +21,57 @@ license = "Elastic License v2"
 name = "Webshell Detection: Script Process Child of Common Web Processes"
```

Comment:
Everywhere else, "web shell" is two words. Use that if possible to rename the file ("Web Shell Detection: Script Process Child of Common Web Processes"